.env

```
@@ -31,6 +31,12 @@ REACT_APP_PROD_HOST_API_URL=http://${WEBSITE_DOMAIN}:${PROD_PORT}
 REACT_APP_VILNACRM_GMAIL=info@vilnacrm.com
 REACT_APP_VILNACRM_PRIVACY_POLICY_URL=https://github.com/VilnaCRM-Org/
 
+# Observability (issue #115). With an empty DSN, Sentry init is a no-op: the SDK is never
+# loaded and local/CI runs stay silent. Populate the DSN, environment, and release in deploys.
+REACT_APP_SENTRY_DSN=
+REACT_APP_SENTRY_ENVIRONMENT=development
+REACT_APP_RELEASE=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The REACT_APP_RELEASE key should go before the REACT_APP_SENTRY_DSN key

(UnorderedKey)